tests/metrics/test_auc.py

```diff
@@ -151,6 +153,81 @@ def test_when_duplicates_in_interactions_insufficient(
         )
         pd.testing.assert_series_equal(metric.calc_per_user(reco, interactions), expected_metric_per_user)
 
+    @pytest.mark.parametrize(
+        "k, insufficient_handling",
+        (
+            (1, InsufficientHandling.IGNORE),
+            (3, InsufficientHandling.IGNORE),
+            (1, InsufficientHandling.EXCLUDE),
+            (3, InsufficientHandling.EXCLUDE),  # user 2 was excluded
+        ),
+    )
+    def test_calc_debias(self, k: int, insufficient_handling: str) -> None:
+        reco = pd.DataFrame(
```

Comment:
Let's do the same structure for test classes and methods as you did for ranking module and stop duplicating a lot of code